src/components/landing/hero/header/hero-hamburger.view.tsx

```diff
@@ -44,17 +37,12 @@ const HeroHamburgerView: React.FC = () => {
           </li>
           <li>
             <div className="hamburger__button">
-              <a
-                href="mailto:sponsor@cruzhacks.com"
+              <p
+                className="hamburger__button-text"
                 style={{ textDecoration: 'none' }}
```

Comment:
this was a no-no on my part back when i first did this. we should move the textDecoration property into the class definition